buildSrc/src/main/groovy/org/apache/beam/gradle/Repositories.groovy

```diff
@@ -0,0 +1,79 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * License); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an AS IS BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.beam.gradle
+
+import org.gradle.api.Project
+
+class Repositories {
```

Comment:
nit: Did you think about using `@CompileStatic` in groovy build plugins? It is little bit more verbose to write the plugin, but on the other it may be more "user friendly" for people that want to contribute to build scripts and don't have a strong groovy / gradle background (it allows then to use auto complete, compile time checks, ...).

http://docs.groovy-lang.org/latest/html/gapi/groovy/transform/CompileStatic.html

Reply:
I didn't know about it. I will definitely use it if it doesn't break Gradle's funky magic. Even better would be moving everything to Kotlin :-)